lib/tachikoma/application.rb

```diff
@@ -221,6 +221,21 @@ def cocoapods
       end
     end
 
+    def npm
+      Dir.chdir("#{Tachikoma.repos_path}/#{@build_for}") do
+        sh(*['git', 'config', 'user.name', @commiter_name])
+        sh(*['git', 'config', 'user.email', @commiter_email])
+        sh(*['git', 'checkout', '-b', "tachikoma/update-#{@readable_time}", @base_remote_branch])
+        sh(*%w(npm install -g npm-check-updates))
```

Comment:
Thanks for your review and maintain a good tool tachikoma.
I see. I'll remove it.